cockpit-frontend/src/components/header/Header.jsx

```diff
@@ -21,6 +26,16 @@ export default function Header() {
             Cockpit
           </IconButton>
           <MvpMenu />
+          <Button
+            variant="outlined"
+            color="primary"
+            className={classes.addButton}
+            onClick={() => {
+              dispatch(show());
```

Comment:
Specify the meaning of the the method, something like showScrumMasterForm could be better